### ISSUE TITLE
Do not separate documentation by empty lines

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1013]
We should report an issue for cases such as following:
```c#
/// <summary>...</summary>

/// <remarks>...</remarks>
public void DoSomething()
{
}
```

Those documentations are considered separate `SingleLineDocumentationCommentTrivia`, due to the empty line between.
They should be merged into one instead (empty lines should be removed):
```c#
/// <summary>...</summary>
/// <remarks>...</remarks>
public void DoSomething()
{
}
```

Reasoning behind:

In C#, XML documentation is closely tied to the structure of the code and is used to generate meaningful documentation through tools like Sandcastle or the built-in IntelliSense in Visual Studio. Avoiding empty lines in XML documentation is recommended because:

- **Parsing Issues:** Empty lines within XML documentation can disrupt parsers or tools that process the comments to generate documentation. These tools expect a clean and continuous XML format, and empty lines might introduce unexpected behavior or errors.

- **Clarity and Consistency:** Removing empty lines ensures that the documentation looks neat and professional. Keeping the comments compact improves readability and prevents unnecessary breaks that can confuse developers reading the code.

- **Efficiency:** Maintaining a consistent style without empty lines reduces clutter and makes it easier to maintain the documentation over time.

By ensuring that XML comments are continuous and well-structured, the documentation becomes more reliable and helpful for both developers and tools.

**Comment by RalfKoban** [comment]
It seems that sometimes, it gets detected and reported by the IDE as `CS1587`.
